contributing-docs/README.rst

```diff
@@ -15,61 +15,58 @@
     specific language governing permissions and limitations
     under the License.
 
-Contributors' guide
+# Contributors' guide
```

Comment:
The `markdown` and `rst` style headers are mixed here. Can you please convert them to .rst style